lib/cdo/i18n_backend.rb

```diff
@@ -151,8 +151,10 @@ def translate(locale, key, options = ::I18n::EMPTY_HASH)
         # sufficient, then refactor the SmartTranslate module so we can use `get_valid_separator` here.
         separator = options[:separator] || ::I18n.default_separator
         # We don't pass in a locale because we want the union of all string keys across all locales.
-        normalized_key = ::I18n.normalize_keys(nil, key, scope, separator).join(separator)
-        I18nStringUrlTracker.instance.log(normalized_key, url, 'ruby') if normalized_key && url
+        # We use -> as the separator in the normalized_key for ease of searching in Crowdin, and to prevent keys
+        # that include a . from getting split in two.
+        normalized_key = ::I18n.normalize_keys(nil, key, scope, ' -> ').join(' -> ')
+        I18nStringUrlTracker.instance.log(normalized_key, url, 'ruby', key, scope, separator) if normalized_key && url
```

Comment:
`normalized` key seems like it could be constructed inside `I18nStringUrlTracker.instance.log` rather than passed in as a parameter. Is there a reason we want the callers to construct it and pass it in?

Reply:
Nope no reason, that's just how it used to be so I hadn't thought to change it. Will be easy enough to move